apps/src/aichat/views/modelCustomization/PromptCustomization.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useState, ChangeEvent} from 'react';
+
+import {StrongText} from '@cdo/apps/componentLibrary/typography/TypographyElements';
+import styles from '../model-customization-workspace.module.scss';
+
+const PromptCustomization: React.FunctionComponent = () => {
+  const [temperature, setTemperature] = useState('0.05');
+
+  const handleTemperatureChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTemperature(event.target.value);
+  };
+
+  return (
+    <div className={styles.verticalFlexContainer}>
+      <div>
+        <div className={styles.inputContainer}>
+          <label htmlFor="chatbot-name">
+            <StrongText>Chatbot name</StrongText>
```

Comment:
Should we start dropping these in `aichat/en_us.json` since we already have that localized file set up? Also ok if we want to hold off until designs are more finalized.

Reply:
Putting them there starts creating work for translators, right? I'd vote to hold off until we have things more solidified